### ISSUE TITLE
Using EventHubTrigger and EventHub Output with .NET Core 2

### ISSUE THREAD

**Issue by hoffe86** [Azure/azure-webjobs-sdk#1453]
Currently we are trying to migrate our EventHub based function (EventHubTrigger & Eventhub output) to Azure Functions v2 (.NET Core 2), but after the refactoring (see Repro Steps) to functions host initialization failed.   

#### Repro steps

1. Replaced old .net 4.7 file with new .net core 2 file.
2. Added all dependencies as nuget packages
3. Try to started the Function from Visaul Studio 

#### Expected behavior
Functions is working like before.

#### Actual behavior

**This error occurs when on host initialization:**
The following 1 functions are in error: EventHubFunction: Unabled to configure binding 'incomingProtobufTimeMgmtMessage' of type 'eventHubTrigger'. This may indicate invalid function.json properties. Can't figure out which ctor to call.

#### Related information 

Provide any related information 
I committed an extract of our Eventhub function implementation to my [personal GitHub repository](https://github.com/hoffe86/personal/tree/master/Azure%20Functions)

**Comment by iremmats** [comment]
Having same issue.
Azure Event Hub trigger in 1.0.7 of Functions and it doesn't find the EventHub trigger at all.

**Comment by EhRom** [comment]
Having the same issue too. 

It may come from the generation of the "function.json" file by the framework, specifically in the bindings section. The path element should be named eventHubName regardless the contructors in the attibutes.

**Comment by jeffhollan** [comment]
Yes problem with the Microsoft.Net.Sdk.Functions serializing the function.json incorrectly.  The "path" should now be "eventHubName" in the new binding extension.

@paulbatum can we bump this next sprint? Blocking for event hub triggers in v2 without a manual edit of the function.json generated in the /bin directory currently.

**Comment by vovikdrg** [comment]
Any progress on this ?

**Comment by fabiocav** [comment]
Opened [this issue](https://github.com/Azure/azure-functions-vs-build-sdk/issues/180) to track the required tooling update.

**Comment by nzthiago** [comment]
@hoffe86  I think this can be closed as the tooling has been updated - can you please confirm?

**Comment by hoffe86** [comment]
@nzthiago bug can be closed. I did a test with newest version of the tooling and it worked!!!
